app/src/main/java/org/simple/clinic/onboarding/OnboardingScreen.kt

```diff
@@ -0,0 +1,70 @@
+package org.simple.clinic.onboarding
+
+import android.content.Context
+import android.support.v4.widget.NestedScrollView
+import android.util.AttributeSet
+import android.widget.Button
+import android.widget.RelativeLayout
+import android.widget.TextView
+import com.jakewharton.rxbinding2.view.RxView
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+import kotterknife.bindView
+import org.simple.clinic.R
+import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.registration.phone.RegistrationPhoneScreen
+import org.simple.clinic.router.screen.RouterDirection
+import org.simple.clinic.router.screen.ScreenRouter
+import org.simple.clinic.util.clamp
+import javax.inject.Inject
+
+
+class OnboardingScreen(context: Context, attributeSet: AttributeSet) : RelativeLayout(context, attributeSet) {
+
+  companion object {
+    @JvmField
+    val KEY = OnboardingScreenKey()
+  }
+
+  @Inject
+  lateinit var router: ScreenRouter
+
+  @Inject
+  lateinit var controller: OnboardingScreenController
+
+  private val appLogo by bindView<TextView>(R.id.onboarding_logo)
+  private val scrollingContainer by bindView<NestedScrollView>(R.id.onboarding_scrolling_container)
+  private val getStartedButton by bindView<Button>(R.id.onboarding_get_started)
+
+  override fun onFinishInflate() {
+    super.onFinishInflate()
+
+    if (isInEditMode) return
```

Comment:
Question: I expect we still are OK with single-line `if` expressions like 
```kotlin
val least = if (a < b) a else b
```
 since they are a Kotlin standard.